etc/jupyter/jupyter_notebook_config.d/jupyterlab.json

```diff
@@ -0,0 +1,7 @@
+{
+  "NotebookApp": {
+    "nbserver_extensions": {
+      "jupyterlab": true
+    }
+  }
+}
```

Comment:
This file should be moved to a `jupyter_server_config.d` directory and configure the `ServerApp`. 
```
{
  "ServerApp": {
    "jpserver_extensions": {
      "jupyterlab": true
    }
  }
}
```